lib/plan.ex

```diff
@@ -0,0 +1,64 @@
+defmodule Braintree.Plan do
+  @moduledoc """
+  Plans represent recurring billing plans in a Braintree merchant account.
+  The API for plans is read only.
```

Comment:
Please add a link to the documentation (as bare as it may be) https://developers.braintreepayments.com/reference/request/plan/all/ruby